src/en/authors-charm-store.md

```diff
@@ -2,204 +2,219 @@ Title: The Juju charm store
 
 # The Juju Charm Store
 
-Juju includes a collection of what we call `Charms` that let you deploy whatever
-services you want in Juju. A collection of charms that are designed to work
-together is called a `Bundle`. Since charms and bundles are open and worked on
-by the community, they represent a distilled set of best practices for deploying
-these services. Both charms and bundles are included in what we collectively
-call The Charm Store.
-
-  - The main project page is here: [https://launchpad.net/charms](https://launchpad.net/charms)
-  - There are useful tools for downloading, modifying, and contributing here: [https://launchpad.net/charm-tools](https://launchpad.net/charm-tools)
-  - Here is the official tutorial for charm authors: [authors-charm-writing](authors-charm-writing.html)
-  - Here is the official tutorial for bundle authors: [charms-bundles](charms-bundles.html)
-
-## Charm Store Submission
-
-There are currently 2 methods to submit a charm and have it listed in the charm
-store. Both methods have their perks - but it is suggested to start with your
-personal namespace before asking for a charmer featured charm.
-
-!!! Note: if you are a member of the Charm Partner Program, you will want to
-ensure that your Charm gets into the Recommended Charms section of the Charm
-Store, so please follow the instructions in the
-[Recommended Charms](#recommended-charms) section below.
+Juju includes a charm store where charms and bundles can be uploaded,
+published, and optionally shared with other users.
 
-## Charm Store Process
+The charm store is broken down into two main sections. Recommended and Community.
+In the Recommended space, charms have been vetted and reviewed by a Juju Charmer
+and all updates to the charm are vetted prior to landing. Community is the wider
+collective of charms shared but not yet submitted or vetted by Juju Charmers.
 
-This process is designed to allow prospective developers to have their charms
-reviewed and updated in the [Charm Store](https://jujucharms.com) in a timely
-manner that ensures peer reviews and quality.
+  - [Charm development information](developer-getting-started.html)
+  - [Bundle creation information](charms-bundles.html)
 
-## Submitting a new Charm
+In order to interact with the charm store you will need the latest
+[charm command](tools-charm-tools.html) and a charm store login.
```

Comment:
For now, to login into charm store, you'll need to have both Ubuntu SSO and Launchpad (LP) account. Due to historical reasons, charm store users the username from LP.